nodejs/week1/src/backend/data/reservations.json

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
Ok you really need to clean up your PRs. Week1 stuff is still here and so is week2. Work on your Git game, one of the most important things you need on any job.